jquery.xblock.js

```diff
@@ -164,33 +164,62 @@
                 },
                 listenTo: function(name, callback) {
                     $this.dispatcher.bind(name, callback);
+                },
+                children: function(element) {
+                    return $(element).data('xblock-children');
+                },
+                childMap: function(block, childName) {
```

Comment:
@e-kolpakov I agree, but the name is part of [the XBlock JavaScript API](http://edx.readthedocs.org/projects/xblock/en/latest/guide/javascript.html#javascript-runtimes), so I can't change it.